scripts/run_e2e_tests.py

```diff
@@ -282,10 +278,6 @@ def build_js_files(dev_mode_setting, deparallelize_terser=False):
         else:
             build.main(args=['--prod_env'])
     else:
-        # The 'hashes.json' file is used by the `url-interpolation` service.
-        if not os.path.isfile(HASHES_FILE_PATH):
-            with python_utils.open_file(HASHES_FILE_PATH, 'w') as hash_file:
```

Comment:
Do we still need to define `HASHES_FILE_PATH`? remove the declaration if it's not required anymore!

Reply:
Yep, removed.